server/src/test/scala/controllers/application/ForecastAccuracyControllerSpec.scala

```diff
@@ -25,7 +31,13 @@ class ForecastAccuracyControllerSpec extends PlaySpec {
   implicit val timeout: akka.util.Timeout = 5.seconds
 
   "ForecastAccuracyController" should {
-    val controller: ForecastAccuracyController = forecastAccuracyController(forecastPax = 120, mlPax = 150, actualPax = 100)
+    val forecastArrivalPax = 150
```

Comment:
should this be 120 ? just looking at previous line it was 120

Reply:
They're for different purposes. I've renamed all variables to hopefully make it clearer what values they're mocking now